docs/Tutorial_Contrast_Profile2.ipynb

```diff
@@ -0,0 +1,474 @@
+{
```

Comment:
<p>Earlier, you mentioned: </p><p> </p><blockquote>Matrix Profile on whole dataset which has&nbsp;<strong>2 similar anomaly behaviors</strong>.</blockquote><p> </p><p>However, you haven't shown result for whole dataset yet. And, then, here, you start explaining <code>T(+)</code> and <code>T(-)</code> . If you think defining T(+) and T(-) is related to the first part, i.e. "Matrix Profile on the first half of dataset which has&nbsp;<strong>1 anomaly behavior</strong>.", then you may create a subsection with an appropriate title. If you think you should talk about 1 and 2 first, then you should have first provided the  results for <code>Matrix Profile on whole dataset</code> , before the definiton of T(+) and T(-).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>